bots/utils.js

```diff
@@ -1,32 +1,72 @@
 "use strict";
 
 const User = require("../models/users.model");
+const Web = require("../models/web.model");
 const config = require("../config/config-loader").load();
 const ovh = require("ovh");
 const Bluebird = require("bluebird");
 const dns = require("dns");
 const URL = require("url");
+const request = require("request");
 
-module.exports = {
-  getOvhClient(senderId) {
-    return User
-      .findOne({ senderId })
-      .exec()
-      .then((userInfos) => {
-        if (!userInfos) {
-          return Bluebird.reject({statusCode: 403, message: "Tu dois d'abord te connecter. Pour ce faire tu peux me le demander"});
+// custom OvhClient for the web users;
+class OvhWebClient {
+  constructor (cookie, userAgent) {
+    this.options = {
+      json: true,
+      encoding: "utf8",
+      headers: {
+        "Content-Type": "application/json; charset=utf-8",
+        Cookie: `cookieAccepted=true; SESSION=${cookie}`,
+        "User-Agent": userAgent,
+        Referer: "https://www.ovh.com/manager/web/",
+        Pragma: "no-cache",
+        "Cache-Control": "no-cache"
+      }
+    };
+  }
+
+  requestPromised (method, path, data) {
+    return new Bluebird((resolve, reject) => {
+      const opt = {
+        method,
+        uri: config.ovh.urlBasePath + path,
+        qs: method === "GET" ? data : {},
+        body: method !== "GET" ? data : {}
+      };
+      return request(Object.assign(opt, this.options), (err, resp, body) => {
+        if (err || resp.statusCode >= 400) {
+          return reject({ statusCode: resp.statusCode, data: body });
         }
+        return resolve(body);
+      });
+    });
+  }
+}
 
-        let ovhClient = ovh({
-          appKey: config.ovh.appKey,
-          appSecret: config.ovh.appSecret,
-          consumerKey: userInfos.consumerKey
+module.exports = {
+  getOvhClient (senderId) {
+    return User.findOne({ senderId }).exec().then((userInfos) => {
+      if (!userInfos) {
+        // check if web user and if so return the custom "ovhClientModule";
+        return Web.findOne({ nichandle: senderId }).exec().then((webClient) => {
+          if (!webClient) {
+            return Bluebird.reject({ statusCode: 403, message: "Tu dois d'abord te connecter. Pour ce faire tu peux me le demander" });
+          }
+          return new OvhWebClient(webClient.cookie, webClient.userAgent);
         });
+      }
 
-        return ovhClient;
+      const ovhClient = ovh({
+        appKey: config.ovh.appKey,
+        appSecret: config.ovh.appSecret,
+        consumerKey: userInfos.consumerKey
       });
+
+      return ovhClient;
```

Comment:
You made the point. There's no comma, but you could add one.

Without the comma, if you add a further row in the future, the `git diff` will show you both lines, the one with `consumerKey...` changed with the additionnal comma, and the real new one.
If you add the comma right now, when you add a row to your object/array, the `git diff` will show you only the real new line.

Just quality-of-life dev ;)